cli/src/commands/operation/diff.rs

```diff
@@ -277,6 +277,42 @@ pub fn show_op_diff(
         }
     }
 
+    let from_wc_ref_targets = get_ordered_wc_ref_targets(from_repo);
+    let to_wc_ref_targets = get_ordered_wc_ref_targets(to_repo);
+    let changed_working_copies: Vec<(&str, (&RefTarget, &RefTarget))> = diff_named_ref_targets(
+        from_wc_ref_targets.iter().map(|(k, v)| (k.as_str(), v)),
+        to_wc_ref_targets.iter().map(|(k, v)| (k.as_str(), v)),
```

Comment:
nit: just `.map(|(k, v)| (k, v))`?

Reply:
SG. I wish there was a prettier way to do this.